applications/asset_tracker_v2/README.rst

```diff
@@ -29,3 +29,4 @@ See the subpages for detailed documentation on the application and its internal
    doc/ui_module
    doc/util_module
    doc/modules_common
+   doc/unit_test
```

Comment:
Not sure if there is a spacing issue here. Please check.
```suggestion
  doc/ui_module
  doc/util_module
  doc/modules_common
  doc/unit_test
```

Reply:
There was no spacing issue. It renders correctly. Not sure why git is showing the spacing issue.